### PR TITLE
venv in gitignore and simplified some logic

### PR DESCRIPTION
**Pull request by noyoshi**
using a map to map (heh) the key to the movement.

also updated the gitignore.

### CODE REVIEW

File: commandChanVim.py [hunk 1]
```diff
@@ -123,26 +130,13 @@ def handleKey(self, key):
...
-                self.body.keypress((rows,cols), 'right')
-
+            if key.isalpha():
+                key = key.lower()
+            
             if key == 'r':
-                pass
+                return
+
+            self.body.keypress((rows, cols), urwidView.KEYMAP[key])
```
Comment (ginglis13) [inline commandChanVim.py]: A KeyError is thrown if a user presses a key not in the map. Can you update to 

```python
try:
    self.body.keypress((rows, cols), urwidView.KEYMAP[key])
except:
    pass
```

or something along those lines?

Comment (noyoshi) [reply commandChanVim.py]: Sounds good

Comment (noyoshi) [reply commandChanVim.py]: Updated @ginglis13

**Comment by wtheisen** [comment]
Welcome to the team we’re really happy to have you onboard! :rocketship:

**Pull request merged**